cmd/snap/cmd_info.go

```diff
@@ -151,6 +152,15 @@ func (x *infoCmd) Execute([]string) error {
 
 		fmt.Fprintf(w, "name:\t%s\n", both.Name)
 		fmt.Fprintf(w, "summary:\t%q\n", both.Summary)
+		fmt.Fprintf(w, "description:\t%q\n", both.Description)
+		if len(both.Apps) > 0 {
```

Comment:
Can we please remove apps from this PR, so we can merge this PR and think a bit further how we want these to be conveyed. I'm pretty sure we want to have something that scales better for more than a trivial number of apps with small sizes, and we probably also want to convey to the user the commands they can use instead of just the app names alone. Couldn't get to an easy agreement with @chipaca today online, so dropping this and merging is likely better than blocking this until we reach agreement.